cmd/build/main.go

```diff
@@ -119,6 +119,12 @@ func resolvePipelineCommits(commitSources map[string][]ostree.SourceSpec) (map[s
 		commitSpecs := make([]ostree.CommitSpec, len(commitSources))
 		for idx, commitSource := range commitSources {
 			var err error
+			commitSource.MTLS = &ostree.MTLS{
+				CA:         os.Getenv("OSBUILD_SOURCES_OSTREE_SSL_CA_CERT"),
+				ClientCert: os.Getenv("OSBUILD_SOURCES_OSTREE_SSL_CLIENT_CERT"),
+				ClientKey:  os.Getenv("OSBUILD_SOURCES_OSTREE_SSL_CLIENT_KEY"),
+			}
+			commitSource.Proxy = os.Getenv("OSBUILD_SOURCES_OSTREE_PROXY")
```

Comment:
In all three development command line utilities, I am leveraging the same ENV variables which are used to do actual ostree pull.

Note that all the code here is only for the commit resolve HTTP(S) request during manifest build.

Reply:
I don't think you need to use any ENV vars in images, take a look at https://github.com/osbuild/images/pull/541/files, something similar should work.